members/M001169.yaml

```diff
@@ -63,78 +66,88 @@ contact_form:
-        - name: mailing_city
-          selector: input#mailing_city
+        - name: Address2
+          selector: input#input-02A611E7-D86B-7139-AD0F-692B62A62CD2
+          value: $ADDRESS_STREET
+          required: false
+        - name: city
+          selector: input#input-02A61225-E09F-EA85-FAF0-D225ED392435
           value: $ADDRESS_CITY
           required: true
-        - name: mailing_zipCode
-          selector: input#mailing_zipCode
+        - name: zipCode
+          selector: input#input-02A612A3-005B-4DF1-0308-92EB04712239
           value: $ADDRESS_ZIP5
           required: true
-        - name: email_address
+        - name: phone
+          selector: input#input-02C9EBF2-9068-15B3-76C7-3DFE96C5D370
+          value: $PHONE
+          required: false
+        - name: email
           selector: input#email
           value: $EMAIL
           required: true
         - name: verify_email
-          selector: input#verify_email
+          selector: input#input-02D398F5-DCDB-34E1-6593-048598518F1D
           value: $EMAIL
           required: true
         - name: subject
-          selector: input#subject
+          selector: input#input-02E0E441-9477-3987-8977-A4EC9BDD0DBD
           value: $SUBJECT
           required: true
         - name: message
-          selector: textarea#message
+          selector: textarea#input-02E45FD5-BD42-AF0E-8834-08C8C725558F
           value: $MESSAGE
           required: true
+          options:
+            max_length: 10000
     - javascript:
         - name: char replace
           values: ["$MESSAGE"]
-          selectors: ["textarea#message"]
+          selectors: ["textarea#input-02E45FD5-BD42-AF0E-8834-08C8C725558F"]
           commands: ["for(var i = 0; i < elements.length; i++) { elements[i].value = values[i].replace(/(%2[0C])/g, '');}"]
           required: true
     - select:
         - name: Your Prefix
-          selector: select#salutation
+          selector: select#input-021E6782-96D4-5780-E206-014F5E35C3F8
           value: $NAME_PREFIX
           required: true
           options:
             Mr.: Mr.
             Ms.: Ms.
             Mrs.: Mrs.
             Miss: Miss
+            Mx.: Mx.
             Dr.: Dr.
             Professor: Professor
             Reverend: Reverend
             Rabbi: Rabbi
             Sister: Sister
             The Honorable: The Honorable
             No Prefix: No Prefix
-    - wait:
-        - value: 1
+    - find:
+        - selector: button.btn
     - click_on:
-        - value: Send
-          selector: input#side-search-btn
+        - value: Submit
+          selector: button.btn
+    - find:
+        - selector: "#form-FE378C8B-ECA5-FC96-D7EE-18EE82AE3ABD h2"
     - find:
-        - selector: "h2"
-          value: "THANK YOU"
+        - selector: "#form-FE378C8B-ECA5-FC96-D7EE-18EE82AE3ABD h2"
   success:
     body:
-      contains: Thanks for taking the time to share your thoughts with me.
+      excludes: Message Topic
```

Comment:
Using exclusively `excludes` is not safe, you must include a `contains`. we can discuss